.github/workflows/static-code-analysis.yml

```diff
@@ -0,0 +1,13 @@
+name: Static code analysis
+
+on: [pull_request]
+
+jobs:
+  static-code-analysis:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run PHPStan
+        uses: docker://jakzal/phpqa:php7.3-alpine
+        with:
+          args: sh -c 'composer install && phpstan analyze src/'
```

Comment:
I have updated jakzal/phpqa to install composer dependencies if it is run as a github action (https://github.com/jakzal/phpqa/pull/185).

As soon as docker hub builds images all you will need is `args: phpstan analyze src/`.

Reply:
Thanks! Then I'll change the action to running phpstan using the minimum required amount of code, and wait until the images are pushed before rerunning the build 👍